src/Backend/Core/Engine/DataGridPaging.php

```diff
@@ -132,21 +132,21 @@ public static function getContent(
             $pagesFirstEnd = 2;
 
             // loop pages
-            for ($i = $pagesFirstStart; $i <= $pagesFirstEnd; $i++) {
+            for ($i = $pagesFirstStart; $i <= $pagesFirstEnd; ++$i) {
```

Comment:
it is,
it is a more efficient way since `$i++` will create a copy and return that increased copy while `++$i` will just update the value of `$i`

```

for($i=0; $i<10; $i++) {
    var_dump($i);
}

for($i=0; $i<10; ++$i) {
    var_dump($i);
}
```

results in 

```
int(0)
int(1)
int(2)
int(3)
int(4)
int(5)
int(6)
int(7)
int(8)
int(9)
int(0)
int(1)
int(2)
int(3)
int(4)
int(5)
int(6)
int(7)
int(8)
int(9)
```